opensearch/src/main/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/relevance/QueryStringQuery.java

```diff
@@ -30,39 +30,45 @@ public class QueryStringQuery extends MultiFieldQuery<QueryStringQueryBuilder> {
    */
   public QueryStringQuery() {
     super(ImmutableMap.<String, QueryBuilderStep<QueryStringQueryBuilder>>builder()
+        .put("allow_leading_wildcard", (b, v) -> b.allowLeadingWildcard(
+            FunctionParameterRepository.convertBoolValue(v, "allow_leading_wildcard")))
         .put("analyzer", (b, v) -> b.analyzer(v.stringValue()))
-        .put("allow_leading_wildcard", (b, v) ->
-            b.allowLeadingWildcard(Boolean.parseBoolean(v.stringValue())))
-        .put("analyze_wildcard", (b, v) ->
-            b.analyzeWildcard(Boolean.parseBoolean(v.stringValue())))
-        .put("auto_generate_synonyms_phrase_query", (b, v) ->
-            b.autoGenerateSynonymsPhraseQuery(Boolean.parseBoolean(v.stringValue())))
-        .put("boost", (b, v) -> b.boost(Float.parseFloat(v.stringValue())))
-        .put("default_operator", (b, v) ->
-            b.defaultOperator(Operator.fromString(v.stringValue())))
-        .put("enable_position_increments", (b, v) ->
-            b.enablePositionIncrements(Boolean.parseBoolean(v.stringValue())))
-        .put("fuzziness", (b, v) -> b.fuzziness(Fuzziness.build(v.stringValue())))
-        .put("fuzzy_rewrite", (b, v) -> b.fuzzyRewrite(v.stringValue()))
-        .put("escape", (b, v) -> b.escape(Boolean.parseBoolean(v.stringValue())))
-        .put("fuzzy_max_expansions", (b, v) ->
-            b.fuzzyMaxExpansions(Integer.parseInt(v.stringValue())))
-        .put("fuzzy_prefix_length", (b, v) ->
-            b.fuzzyPrefixLength(Integer.parseInt(v.stringValue())))
-        .put("fuzzy_transpositions", (b, v) ->
-            b.fuzzyTranspositions(Boolean.parseBoolean(v.stringValue())))
-        .put("lenient", (b, v) -> b.lenient(Boolean.parseBoolean(v.stringValue())))
-        .put("max_determinized_states", (b, v) ->
-            b.maxDeterminizedStates(Integer.parseInt(v.stringValue())))
+        .put("analyze_wildcard", (b, v) -> b.analyzeWildcard(
+            FunctionParameterRepository.convertBoolValue(v, "analyze_wildcard")))
+        .put("auto_generate_synonyms_phrase_query", (b, v) -> b.autoGenerateSynonymsPhraseQuery(
+            FunctionParameterRepository.convertBoolValue(v,"auto_generate_synonyms_phrase_query")))
```

Comment:
```suggestion
            FunctionParameterRepository.convertBoolValue(v, "auto_generate_synonyms_phrase_query")))
```

Reply:
If I add the space, line pass 100 symbols limit and I have to break it. I would like to avoid it, but if you insist...